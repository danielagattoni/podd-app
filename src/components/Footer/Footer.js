import React from 'react';

const Footer = () => (
  <footer className="Footer">
    <p>Footer</p>
  </footer>
);

export default Footer;
