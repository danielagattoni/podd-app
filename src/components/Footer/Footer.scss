@import "../../styles/config.mixins.scss";
@import "../../styles/config.variables";

.Footer {
  grid-column: 1 / -1;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  align-content: center;
  text-align: center;
  min-height: 10rem;
  font-size: map-get($font-size-primary, xxs);

  @include break(md) {
    flex-direction: row;
  }
}
