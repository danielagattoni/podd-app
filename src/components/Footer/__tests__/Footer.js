import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from 'components/Footer/Footer.js';

const renderer = new ShallowRenderer();

describe('Footer component', () => {

  it('displays Footer component with copyrights', () => {
    const component = renderer.render(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
