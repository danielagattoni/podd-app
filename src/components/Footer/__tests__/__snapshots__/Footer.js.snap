// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component displays Footer component with copyrights 1`] = `
<footer
  className="Footer"
>
  <div
    className="Container"
  >
    <p
      className="Footer-copyright"
    >
      Made with â™¥ by Daniela Gattoni, Frontend Engineer.
    </p>
  </div>
</footer>
`;
